src/csharp/GenericTests.cs

```diff
@@ -4,7 +4,7 @@
 using System.Text;
 using NUnit.Framework;
 
-#if NETCOREAPP1_1
+#if NETCOREAPP1_1 || NETCOREAPP2_0
 namespace NUnitCoreTestDemo
 #else
 namespace NUnitTestDemo
```

Comment:
I'm inclined distinction to remove this in all the source files. There doesn't seem to be any reason to use a different namespace for .NET Core.

Reply:
Yes, this was just the way it was originally from @rprouse .

Reply:
Agree with the naming convention

Reply:
I used the different namespaces so that when you are testing in Visual Studio, you can see which tests are for which platform and group them accordingly.

Reply:
You mean when running multiple platforms at the same time then?  But does that really work yet?

Reply:
I might be mis-remembering, but didn't we have multiple projects using the same files? The multiple projects allowed us to test each platform, but you are right, multi-targeting from one project still isn't working.

Reply:
Just tried it, netcore2.0 and net461 at the same time, and I get tests for the netcore, but also error messages, and no tests for net461.  I believe we have to wait to see this working, for some later VS version.

Reply:
I split things so there is one solution per project. For most hands-on testing, I prefer that. When we get to multiple project testing, we can create more solutions. 

VS let's you order tests by project so I don't see that the namespace is essential.